queries/orderbook/barn_batch_rewards.sql

```diff
@@ -78,12 +81,14 @@ order_protocol_fee AS (
         os.auction_id,
         os.solver,
         os.tx_hash,
+        os.order_uid,
         os.sell_amount,
         os.buy_amount,
         os.sell_token,
         os.observed_fee,
         os.surplus,
         os.surplus_token,
+        convert_from(os.app_data, 'UTF8')::JSONB->'metadata'->'partnerFee'->>'recipient' as protocol_fee_recipient,
```

Comment:
Before merging: rename to `partner_fee_recipient`. Also add `protocol_fee_kind` here to be consistent with dune-sync.

Reply:
Just pushed a change for this. Let me know if that looks good.